week-4/implement/get-ordinal-number.test.js

```diff
@@ -1,4 +1,18 @@
 // In this week's prep, we started implementing getOrdinalNumber
+function getOrdinalNumber(num) {
+  if (num % 10 === 1) {
+    if (num === 11) {
+      return "11th";
+    }
+    return `${num}st`;
+  }
+}
+
+test("works for any number ending in 1", function () {
+  expect(getOrdinalNumber(1)).toBe("1st");
+  expect(getOrdinalNumber(11)).toBe("11th");
+  expect(getOrdinalNumber(21)).toBe("21st");
+});
```

Comment:
I did mine a bit differently. Mine is a bit more concise, but this makes sense as well. Well done!